content/post/082-our-focus-for-2020.md

```diff
@@ -0,0 +1,86 @@
+
+Excited to use these new improvements to IPFS in your package management use case? [Make an issue in the package managers repo](https://github.com/ipfs/package-managers/issues) for the latest recommendations!
+
+## 2020 Roadmapping Process
+
+We did a slightly different roadmapping process this year, after learning from the wins and challenges we encountered with our massive roadmapping endeavor in 2019. Rather than starting with project goals to inspire a distributed working group roadmapping process (which was difficult to merge and hard to make open), we decided to flip the order of things and start with a [broad call for themes and suggestions](https://github.com/ipfs/roadmap/blob/master/2020-IPFS-Project-Planning.md) from the community for where the IPFS project should focus in 2020. 
+
+We got [**11** awesome proposals](https://github.com/ipfs/roadmap/issues), with great ideas and discussion across a wide variety of tracks. Notable mentions include [IPFS in Rust](https://github.com/ipfs/roadmap/issues/54), [IPFS for Mobile](https://github.com/ipfs/roadmap/issues/45), [IPFS-Wikipedia](https://github.com/ipfs/roadmap/issues/46), and [IPFS-powered GitHub](https://github.com/ipfs/roadmap/issues/43). Huge thank you for the community members who took the time to write these proposals, and call out the core needs and gaps, which fed directly into our next stage!
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Huge thanks to <a href="https://twitter.com/agentofuser?ref_src=twsrc%5Etfw">@agentofuser</a> <a href="https://twitter.com/dietrich?ref_src=twsrc%5Etfw">@dietrich</a> <a href="https://twitter.com/Ianopolous?ref_src=twsrc%5Etfw">@Ianopolous</a> <a href="https://twitter.com/lidelOrg?ref_src=twsrc%5Etfw">@lidelOrg</a> <a href="https://twitter.com/NukeManDan?ref_src=twsrc%5Etfw">@NukeManDan</a> <a href="https://twitter.com/aschmahmann?ref_src=twsrc%5Etfw">@aschmahmann</a> <a href="https://twitter.com/vasa_develop?ref_src=twsrc%5Etfw">@vasa_develop</a> &amp; <a href="https://twitter.com/hashtag/liyunzhi_666?src=hash&amp;ref_src=twsrc%5Etfw">#liyunzhi_666</a> for the amazing contributions. There’s far more here than one team can build in 2020, but all awesome ideas &amp; much we can empower the whole community to pitch in on! <a href="https://t.co/jUmEJ6nd3k">pic.twitter.com/jUmEJ6nd3k</a></p>&mdash; Molly (@momack28) <a href="https://twitter.com/momack28/status/1193404771983454214?ref_src=twsrc%5Etfw">November 10, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
11 proposals noted above, but only 8 ppl listed here - if listing, should list all (including you!)

Reply:
rust and offline-first came after this tweet (https://twitter.com/momack28/status/1193404771983454214) - I can fake the rewrite here, but I can't change the actual tweet...

Reply:
```suggestion
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Huge thanks to <a href="https://twitter.com/agentofuser?ref_src=twsrc%5Etfw">@agentofuser</a> <a href="https://twitter.com/dietrich?ref_src=twsrc%5Etfw">@dietrich</a> <a href="https://twitter.com/Ianopolous?ref_src=twsrc%5Etfw">@Ianopolous</a> <a href="https://twitter.com/lidelOrg?ref_src=twsrc%5Etfw">@lidelOrg</a> <a href="https://twitter.com/NukeManDan?ref_src=twsrc%5Etfw">@NukeManDan</a> <a href="https://twitter.com/aschmahmann?ref_src=twsrc%5Etfw">@aschmahmann</a> <a href="https://twitter.com/vasa_develop?ref_src=twsrc%5Etfw">@vasa_develop</a> <a href="https://twitter.com/hashtag/liyunzhi_666?src=hash&amp;ref_src=twsrc%5Etfw">#liyunzhi_666</a> <a href="https://twitter.com/hashtag/gozala?src=hash&amp;ref_src=twsrc%5Etfw">@gozala</a> &amp; <a href="https://twitter.com/hashtag/parkan?src=hash&amp;ref_src=twsrc%5Etfw">@parkan</a> for the amazing contributions. There’s far more here than one team can build in 2020, but all awesome ideas &amp; much we can empower the whole community to pitch in on! <a href="https://t.co/jUmEJ6nd3k">pic.twitter.com/jUmEJ6nd3k</a></p>&mdash; Molly (@momack28) <a href="https://twitter.com/momack28/status/1193404771983454214?ref_src=twsrc%5Etfw">November 10, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```